aws/runner-machine-image.pkr.hcl

```diff
@@ -0,0 +1,48 @@
+packer {
+  required_plugins {
+    amazon = {
+      version = ">=0.0.2"
+      source  = "github.com/hashicorp/amazon"
+    }
+  }
+}
+
+source "amazon-ebs" "runner_machine_image" {
+  ami_name      = format("%s-ghr%s-nv%s", regex_replace(var.source_image, "(ubuntu/images/\\*)|(-\\*)|(\\.)", "" ), replace(var.runner_version, ".", ""), replace(var.nvidia_version, ".", ""))
+  instance_type = var.instance_type
+  region        = var.region
+  source_ami_filter {
+    filters = {
+      name                = var.source_image
+      root-device-type    = "ebs"
+      virtualization-type = "hvm"
+    }
+    most_recent = true
+    owners      = ["099720109477"]
+  }
+  ssh_username = "ubuntu"
+  run_tags = {
```

Comment:
You can remove `run_tags`

Reply:
If I do that the default tag would be "Packer Builder"